<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Transmission</string>
     </attribute>
     <widget class="QComboBox" name="ctl_cbMSGType">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>450</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ctl_btnSend">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="ctl_tableFCS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>601</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ctl_btnAutoTest">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>450</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>FCS</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ctl_IP_FCS">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ctl_portFCS">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ctl_btnApplyFCSConf">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ctl_btnCancelFCSConf">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>231</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>BCC_B6000</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ctl_IP_BCC">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ctl_portBCC">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ctl_btnApplyBCCConf">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ctl_btnCancelBCCConf">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
